### DISCUSSION TITLE
Infinite Scroll in Select, MultiSelect, etc.

### DISCUSSION THREAD

**Discussion by ibadkhan1996** [mantinedev/mantine#5825] [Feature requests]
![preview](https://github.com/mantinedev/mantine/assets/48190153/a17dafca-cf4f-497a-b529-1d55c8302766)
I want to get access of ScrollArea props in Mantine components which have dropdowns so that I can get use of the ScrollArea and create infinite scroll on scroll end using hooks and tanstack-query

**Comment by rtivital** [comment]
You can get full control over the rendering by building a custom component with Combobox

**Reply by ibadkhan1996** [reply]
Sure, thanks! :)
